docs/site/tutorials/python_interoperability.ipynb

```diff
@@ -1,608 +1,668 @@
+    "// See the License for the specific language governing permissions and\n",
+    "// limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "sI1ZtrdiA4aY"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/swift/tutorials/python_interoperability\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/swift/blob/master/docs/site/tutorials/python_interoperability.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/swift/blob/master/docs/site/tutorials/python_interoperability.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8sa42_NblqRE"
+   },
+   "source": [
+    "# Python interoperability\n",
+    "\n",
+    "Swift For TensorFlow supports Python interoperability.\n",
+    "\n",
+    "You can import Python modules from Swift, call Python functions, and convert values between Swift and Python."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Python Versions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, on `import Python`, Swift searches system library paths for the newest version of Python installed. \n",
+    "To use a specific Python installation, set the `PYTHON_LIBRARY` environment variable to the `libpython` shared library provided by the installation.  For example: \n",
+    "\n",
+    "`export PYTHON_LIBRARY=\"~/anaconda3/lib/libpython3.7m.so\"`\n",
+    "\n",
+    "The exact filename will differ across Python environments and platforms."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
     "colab": {
-      "name": "Python interoperability.ipynb",
-      "version": "0.3.2",
-      "provenance": [],
-      "collapsed_sections": [],
-      "toc_visible": true
+     "base_uri": "https://localhost:8080/",
+     "height": 51
     },
-    "kernelspec": {
-      "name": "swift",
-      "display_name": "Swift"
+    "colab_type": "code",
+    "id": "kZRlD4utdPuX",
+    "outputId": "101c8d8f-9d75-485f-9678-d27ece433fbb"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3.6.7 (default, Oct 22 2018, 11:32:17) \r\n",
+      "[GCC 8.2.0]\r\n"
+     ]
     }
+   ],
+   "source": [
```

Comment:
Could you please move this code block under the title section, and remove the `// export ...` line? I hope this picture makes things clear:

![image](https://user-images.githubusercontent.com/5590046/55294423-d7221880-53b6-11e9-954c-355d4bbbdfe2.png)

Reply:
+1.